flink-cdc-runtime/src/main/java/org/apache/flink/cdc/runtime/operators/transform/PostTransformOperator.java

```diff
@@ -539,6 +541,10 @@ private void initializeUdf() {
                             // into UserDefinedFunction interface, thus the provided UDF classes
                             // might not be compatible with the interface definition in CDC common.
                             Object udfInstance = udfFunctionInstances.get(udf.getName());
+                            if (udfInstance instanceof BuiltInModel) {
+                                ((BuiltInModel) udfInstance)
+                                        .configure(Configuration.fromMap(udf.getParameters()));
```

Comment:
Supporting configuration is also a feature for udf. The method `configure` should be in the interface `UserDefinedFunction` instead of `BuiltInModel`.

Reply:
done.
Removed BuiltInModel and moved this interface to  UDF.